crates/fdev/src/commands.rs

```diff
@@ -143,32 +139,49 @@ pub async fn update(config: UpdateConfig, other: BaseConfig) -> Result<(), anyho
         StateDelta::from(buf).into()
     };
     let request = ContractRequest::Update { key, data }.into();
-    execute_command(request, other).await
+    execute_command(request, other, config.address, config.port).await
 }
 
 async fn execute_command(
     request: ClientRequest<'static>,
     other: BaseConfig,
+    address: IpAddr,
+    port: u16,
 ) -> Result<(), anyhow::Error> {
     let mode = other.mode;
-    let paths = other.paths.build()?;
-    let contract_store = ContractStore::new(paths.contracts_dir(mode), DEFAULT_MAX_CONTRACT_SIZE)?;
-    let delegate_store = DelegateStore::new(paths.delegates_dir(mode), DEFAULT_MAX_DELEGATE_SIZE)?;
-    let secret_store = SecretsStore::new(paths.secrets_dir(mode))?;
-    let state_store = StateStore::new(Storage::new(&paths.db_dir(mode)).await?, MAX_MEM_CACHE)?;
-    let rt =
-        freenet::dev_tool::Runtime::build(contract_store, delegate_store, secret_store, false)?;
-    let mut executor = Executor::new(state_store, || Ok(()), OperationMode::Local, rt, None)
```

Comment:
so we are not using the executor directly from dev in other commands? let's update those too